### PR TITLE
[Improvement] Empty State Components Added to Request & Permit Holder Tables

### PR DESCRIPTION
**Pull request by aaggarwal10**
## Notion ticket link

[Add empty states for Request & Permit Holder Tables](https://www.notion.so/uwblueprintexecs/ac7b1f636a214a0cbf45f6b76f899b54?v=5414ee5fc5974f149921e192d6eb29b8&p=9d22a3cb787e44e0ac68d5888881f9d4)

## Implementation description
* Added Empty States to Permit Holder & Request Pages
* Permit Holder Page:
image
* Request Page:
image

## Notes
* Extracted Component from `reports.tsx` adding a custom component in `components/admin` to be used in Request, Permit Holder, and Reports Page. 

## Checklist
- [X] My PR name is descriptive, is in imperative tense and starts with one of the following: `[Feature]`,`[Improvement]` or `[Fix]`,
- [X] I have run the appropriate linter(s)
- [X] I have requested a review from the RCD team on GitHub, or specific people who are associated with this ticket

### CODE REVIEW

File: components/admin/EmptyStateComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import { Box, Text } from '@chakra-ui/react'; // Chakra UI
```
Comment (OustanDing) [inline components/admin/EmptyStateComponent.tsx:1]: Please move and rename this file to `components/EmptyMessage.tsx`

File: components/admin/EmptyStateComponent.tsx [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+import { SearchIcon } from '@chakra-ui/icons'; // Chakra UI icons
+
+type EmptyStateComponentProps = {
+  readonly titleMessage: string;
+  readonly subMessage: string;
```
Comment (OustanDing) [inline components/admin/EmptyStateComponent.tsx:6]:
```suggestion
  readonly title: string;
  readonly message: string;
```

File: components/admin/EmptyStateComponent.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+};
+
+export default function EmptyStateComponent({
+  titleMessage,
+  subMessage,
+}: EmptyStateComponentProps) {
+  return (
+    <Box padding="120px">
+      <SearchIcon w={20} h={20} color="#8C9196" />
+      <Text padding="12px 0 12px" textStyle="display-large">
```
Comment (OustanDing) [inline components/admin/EmptyStateComponent.tsx:16]:
```suggestion
      <Text padding="12px 0" textStyle="display-large">
```

**Review by OustanDing — CHANGES_REQUESTED** [review]

**Comment by carelynntsai** [comment]
I can't formally approve this because of the merge conflicts but it looks good from my end

File: pages/admin/index.tsx [hunk 4]
```diff
@@ -43,6 +43,7 @@ import useDebounce from '@tools/hooks/useDebounce'; // Debounce hook
 import { Column } from 'react-table';
 import { formatDateVerbose, formatFullName } from '@lib/utils/format'; // Verbose date formatter util
 import GenerateReportModal from '@components/admin/requests/reports/GenerateModal'; // Generate report modal
+import EmptyStateComponent from '@components/EmptyMessage';
```
Comment (OustanDing) [inline pages/admin/index.tsx:46]: Could you please change all imported names for this component to `EmptyMessage` now?

**Review by OustanDing — CHANGES_REQUESTED** [review]

**Review by OustanDing — APPROVED** [review]
LGTM!

**Pull request merged**